packages/ui/components/core/src/browserDetection.js

```diff
@@ -36,4 +36,12 @@ export const browserDetection = {
   isIOSChrome: checkChrome('ios'),
   isChromium: checkChrome('chromium'),
   isMac: navigator.appVersion.indexOf('Mac') !== -1,
+  isIOS: /iPhone|iPad|iPod/i.test(navigator.userAgent),
+  isMacSafari:
```

Comment:
Yes, now it is used.

I saw this code in the OverlaysManager with a TODO that says that it would be good to move this code to core's browserDetection so I moved it and modified it together with their tests.